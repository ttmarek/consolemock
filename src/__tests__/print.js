const { makeConsoleMock } = require('../consolemock');

describe('PRINT', () => {
  test('calling .print() incorrectly throws a helpful error message', () => {
    const mock = makeConsoleMock();
    const callPrintIncorrectly = () => {
      mock.print();
    };

    expect(callPrintIncorrectly).toThrowErrorMatchingSnapshot();
  });
});
