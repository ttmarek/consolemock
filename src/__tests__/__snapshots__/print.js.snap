// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PRINT calling .print() incorrectly throws a helpful error message 1`] = `
"[consolemock]
You called .print without giving makeConsoleMock a native console object:
  import { makeConsoleMock } from 'consolemock';
  console = makeConsoleMock(console);"
`;
